docs/source/models/concept_models/vivarium_mncnh_portfolio/concept_model.rst

```diff
@@ -62,43 +62,122 @@ and details on the intrapartum and neonatal time periods.
 3.0 Concept model and submodels
 +++++++++++++++++++++++++++++++
 
-.. todo::
-
-  Insert colorful full concept model
-
-.. _mncnh_portfolio_3.1:
-
-3.1 Model Waves
----------------
+We plan to complete this work in 3 waves. 
 
-**Wave 1**: Add new risk and cause models and update existing risks and causes as needed
+* Wave 1 will include in basic model design, outlines of the healthcare system, and some interventions (AI ultrasound, higher level delivery facility interventions, RDS management). 
+* Wave 2 will some antenatal supplements(MMS, IV iron), the hemoglobin risk for birthing parents, and all downstream causes affected by hemoglobin. 
+* Wave 3 will add in gestational blood pressure and relevant causes and risks including pre-eclampsia care and downstream effects of high blood pressure. 
 
-.. todo::
+**Wave 1 Concept Model Map:**
 
-  Determine if wave 1 will include making multiple, smaller simulations covering a subset of risks and causes or if it will all be integrated.
+.. image:: wave_1_full.svg
```

Comment:
Is there some way to make this diagram narrower and more legible in-line? I have to open it in a new tab and scroll back and forth to read it...